examples/wallet/script/src/bin/execute.rs

```diff
@@ -124,7 +123,7 @@ mod tests {
 
   #[test]
   fn deploy_template() {
-    athena_sdk::utils::setup_logger();
+    env_logger::builer().is_test(true).try_init();
```

Comment:
`builer` here looks like a typo

Reply:
Yes, thanks! We really should start building and running the /examples and /tests in CI.